### DISCUSSION TITLE
It's possible to use netmaker without behind NAT without a public ip ?

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [gravitl/netmaker#1260] [Q&A] [answered]
Hi,

i'm on hetzner so public ip(s) are limited and say i've 1 shared public ip (natted)  and 443 port already busy .

It's possible to use netmaker without a proxy server as standard configuration connecting directly to http port of netmaker (http port different than 443 ) ?

Thanks.

**Comment by mattkasun** [comment, marked as answer]
Yes, you can setup netmaker to just use http ports, but not recommended
